### PR TITLE
feat(starfish): add table for transaction epm and p75 in db query details

### PR DESCRIPTION
**Pull request by DominikB2014**
This PR adds two graphs above the transaction table in the db query details page.
1. EPM of the top 5 transactions in the table.
2. P75 of the top 5 transaction in the table.

### CODE REVIEW

File: static/app/views/starfish/modules/databaseModule/panel/index.tsx [hunk 1]
```diff
@@ -238,6 +248,27 @@ function QueryDetailBody({
...
         </FlexRowItem>
+        <FlexRowItem>
+          <SubHeader>{t('Top 5 Transactions by P75')}</SubHeader>
+          <Chart
+            statsPeriod="24h"
+            height={140}
+            data={p75TransactionSeries}
+            start=""
+            end=""
+            loading={isDataLoading}
```
Comment (0Calories) [inline static/app/views/starfish/modules/databaseModule/panel/index.tsx:259]: Did you forget to update this?
```suggestion
            loading={isP75GraphLoading}
```

File: static/app/views/starfish/modules/databaseModule/queries.ts [hunk 2]
```diff
@@ -380,15 +380,20 @@ export const useQueryMainTable = (options: {
...
+  {count: number; interval: string; p75: number; transaction: string}[]
+> => {
   const pageFilter = usePageFilters();
   const {startTime, endTime} = getDateFilters(pageFilter);
   const dateFilters = getDateQueryFilter(startTime, endTime);
-  const queryFilter = `group_id = '${row.group_id}'`;
+  const queryFilter = `transaction IN ('${transactionNames.join(`', '`)}')`;
 
   const query = `
   select
```
Comment (0Calories) [inline static/app/views/starfish/modules/databaseModule/queries.ts:394]: big nit lol but I see we're switching the case of operations in these queries, i.e `SELECT` vs `select` and we should stick to one or the other

**Review by 0Calories — APPROVED** [review]
💥

File: static/app/views/starfish/modules/databaseModule/queries.ts (hunk 2 above)
Comment (DominikB2014) [reply static/app/views/starfish/modules/databaseModule/queries.ts:394]: Yeah 👍, its a fair point, we're all over the places with these queries, i'll update them all at once in another PR.

File: static/app/views/starfish/modules/databaseModule/panel/index.tsx (hunk 1 above)
Comment (DominikB2014) [reply static/app/views/starfish/modules/databaseModule/panel/index.tsx:259]: Yes I did! ty, nice catch!

**Pull request merged**